visualizer/src/Controls/ImageControls/RawControls/GrayscaleControls.js

```diff
@@ -0,0 +1,216 @@
+import { FormLabel, makeStyles } from '@material-ui/core';
+import FormControlLabel from '@material-ui/core/FormControlLabel';
+import FormGroup from '@material-ui/core/FormGroup';
+import Grid from '@material-ui/core/Grid';
+import Select from '@material-ui/core/Select';
+import Slider from '@material-ui/core/Slider';
+import Switch from '@material-ui/core/Switch';
+import Tooltip from '@material-ui/core/Tooltip';
+import { useSelector } from '@xstate/react';
+import React, { useEffect, useRef } from 'react';
+import { useRaw } from '../../../ServiceContext';
+
+const useStyles = makeStyles(theme => ({
+  root: {
+    paddingTop: theme.spacing(1),
+  },
+}));
+
+const InvertToggle = ({ channel }) => {
+  const invert = useSelector(channel, state => state.context.invert);
+
+  // Adds mousetrap class so hotkeys work after using switch
+  const inputRef = useRef();
+  useEffect(() => {
+    const input = inputRef.current;
+    input.className = input.className + ' mousetrap';
```

Comment:
Prefer the string formatting over concatenation:

```suggestion
    input.className = `${input.className}  mousetrap`;
```